credentials.json

```diff
@@ -0,0 +1,10 @@
+{
+  "web": {
+    "client_id": "294321754695-t48b7ubomafqcitnmve2ngeotg4npsd2.apps.googleusercontent.com",
+    "project_id": "google.com:spontaneously",
+    "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+    "token_uri": "https://oauth2.googleapis.com/token",
+    "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+    "client_secret": "ByF-DdMdKvO-jby_3TfADwDx"
```

Comment:
Now that this file is in this PR this is no longer a secret. A malicious person could find this PR and use this key to pretend to be the Ad-Lib project. Please remove this file from the PR and **generate a new secret**. Some useful info:
https://cloud.google.com/docs/authentication/api-keys#securing_an_api_key

Reply:
I have added credentials.json to gitignore